Libraries/MSSQLDevice/Source/Device.MSSQL/MSSQLTypes.cs

```diff
@@ -446,11 +450,83 @@ public class MSSQLBinary : SQLBinary
 
 		protected override string InternalNativeDomainName(MetaData AMetaData)
 		{
-			return "image";
+			return "varbinary(max)";

```

Comment:
Also note that this would only be an issue for someone using the auto-reconciliation process, so it isn't likely to actually break anyone.